hazelcast/src/main/java/com/hazelcast/client/console/HazelcastCommandLine.java

```diff
@@ -817,6 +818,17 @@ private static boolean isActive(JobStatus status) {
         return status != JobStatus.FAILED && status != JobStatus.COMPLETED;
     }
 
+    private static HazelcastClientInstanceImpl getHazelcastClientInstanceImpl(HazelcastInstance client) {
+        if (client instanceof HazelcastClientProxy) {
+            return ((HazelcastClientProxy) client).client;
+        } else if (client instanceof HazelcastClientInstanceImpl) {
+            return ((HazelcastClientInstanceImpl) client);
+        } else {
+            throw new IllegalArgumentException("This method can be called only with client"
+                    + " instances such as HazelcastClientProxy and HazelcastClientInstanceImpl.");
+        }
+    }
```

Comment:
I couldn't see `Util` class in hazelcast package to move them into so I prefer to keep it in `c.h.jet.impl.util.Util` for now. I thought that if I created a Util class to store this static methods in hazelcast package, it would exceed the scope of this pr. For this `getHazelcastClientInstanceImpl`, I didn't define it in `c.h.jet.impl.util.Util` because I don't want to access jet package from client package. Anyway, I think these should be discussed in a separate pr.